manifests/r/RubyInstallerTeam/Ruby/3/1/3.1.1-1/RubyInstallerTeam.Ruby.3.1.installer.yaml

```
@@ -18,8 +18,16 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-3.1.1-1/rubyinstaller-3.1.1-1-x64.exe
   InstallerSha256: 130BEA0AEC11B56EE1AF8BA443BCB9E44FE806AA20AC996CA49158FA2BE870B8
+  AppsAndFeaturesEntries:
+    DisplayName: Ruby 3.1.1-1-x64-ucrt
+    Publisher: RubyInstaller Team
+    ProductCode: RubyInstaller-3.1-x64-mingw-ucrt_is1
```

```suggestion
  - DisplayName: Ruby 3.1.1-1-x64-ucrt
    ProductCode: RubyInstaller-3.1-x64-mingw-ucrt_is1
```
Publisher is not needed as it matches in the Locale file